phys/module_sf_sfclayrev.F

```diff
@@ -1103,9 +1134,74 @@ SUBROUTINE sfclayrevinit
        psih_unstab(n)=psih_unstable_full(zolf)
 
     ENDDO
+    IF ( shalwater_rough .EQ. 1 ) THEN
+       CALL shalwater_init(ims,ime,jms,jme,            &
+                 its,ite,jts,jte,                      &
+                 bathymetry_flag, shalwater_rough,     &
+                 shalwater_depth, water_depth,         &
+                 xland,LakeModel,lake_depth,lakemask   )
+    END IF
 
    END SUBROUTINE sfclayrevinit
 
+   SUBROUTINE shalwater_init(ims,ime,jms,jme,                    &
+                             its,ite,jts,jte,                    &
+                             bathymetry_flag, shalwater_rough,   &
+                             shalwater_depth, water_depth,       &
+                             xland,LakeModel,lake_depth,lakemask )
+
+   INTEGER, INTENT(IN)      ::   ims,ime,jms,jme,its,ite,jts,jte
+   INTEGER, INTENT(IN)      ::   shalwater_rough 
+   REAL,    INTENT(IN)      ::   shalwater_depth 
+   INTEGER, INTENT(IN)      ::   bathymetry_flag 
+   REAL,    DIMENSION( ims:ime, jms:jme ), INTENT(INOUT)    ::  water_depth
+   REAL,    DIMENSION( ims:ime, jms:jme ), INTENT(IN   )    ::  xland 
+   INTEGER         ::     LakeModel
+   REAL,    DIMENSION( ims:ime, jms:jme ), INTENT(IN   )    ::  lake_depth
+   REAL,    DIMENSION( ims:ime, jms:jme )                   ::  lakemask  
+
+   ! Local 
+   LOGICAL :: overwrite_water_depth
+
+   overwrite_water_depth = .False.
+
+   IF ( bathymetry_flag .eq. 1 ) THEN
+      IF ( shalwater_depth .LE. 0.0 ) THEN
+         CALL wrf_message ( " Bathymetry dataset from GEBCO Compilation Group. Please acknowledge the following in presentations and publications: GEBCO Compilation Group (2021) GEBCO 2021 Grid (doi:10     .5285/c6612cbe-50b3-0cff-e053-6c86abc09f8f)." )
```

Comment:
@phawbeck Should this be in the code or added to the release note? We don't have any other prints from the code to do this.

Reply:
@phawbeck I see that this message is in the README file. I think adding it to the RELEASE NOTE should be sufficient.

Reply:
I don't feel strongly one way or the other, so whatever you recommend! Thanks!

Reply:
@phawbeck Can you then remove this print?

Reply:
@phawbeck Can you remove this message, as we discussed?

Reply:
Done!